backend/handler.js

```diff
@@ -39,13 +39,15 @@ module.exports.run = async event => {
     if (!body || body.query === "") throw Error("missing_body");
     
     const connection = await sql.connect(db(group));
-    const db_response = await sql.query(queryPagination(body.query,body.pageNumber,pageSize));
-    
-    if(body.query.toUpperCase().split(' ')[0].includes('SELECT')){
-      db_response_count = await sql.query(countQueryPagination(body.query));
-      console.log('db_response_count',db_response_count)
-      db_response.rowsAffected = [db_response_count.recordset[0].row_count]
-    }
+    const db_response = await sql.query(body.query);
+  
+    //THESE METHODS ARE FOR THE PAGINATION OF THE INFORMATION RETURNED BY THE QUERIES.
```

Comment:
remove this comments, as they are not used

Reply:
done